packages/block-editor/src/components/fullscreen-mode/style.scss

```diff
@@ -23,7 +22,6 @@ body.js.is-fullscreen-mode {
 			transition: left 0.12s ease-in-out;
 		}
 
-		// This is all so stupid
```

Comment:
🤣

Reply:
lol, I often make a lot of comments as I'm working on code. Most get deleted. They usually include much more... harsh... language. 😆

Reply:
😂